posts/2023-05-02-23.0.0.4.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "ARM64 container images in Open Liberty 23.0.0.4"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: ARM64 container images in Open Liberty 23.0.0.4 - OpenLiberty.io
+seo-description: Open Liberty 23.0.0.4 provides better, more consistent logging for the features with which the runtime is running.  Joining the existing AMD64, PPC64LE, and S390X architectures, this release also provides container images for the ARM64 architecture.  This release also includes an important security vulnerability (CVE) fix and many notable bug fixes.
```

Comment:
```suggestion
seo-description: Open Liberty 23.0.0.4 provides better, more consistent logging for the features that are installed on the runtime. Starting with this release, container images for the ARM64 architecture are available, along with the existing AMD64, PPC64LE, and S390X architectures. This release also includes an important security vulnerability (CVE) fix and many notable bug fixes.
```

Reply:
Pulling in the second part only as the first time is specific to 'running' features, not 'installed' features.